C:\Users\thats\OneDrive\COS470\Project>python generate_model.py
2021-04-29 20:58:47.240119: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll2.4.1
2021-04-29 20:58:55.599658: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-04-29 20:58:55.605348: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library nvcuda.dll
2021-04-29 20:58:55.659790: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties: 
pciBusID: 0000:01:00.0 name: GeForce RTX 3060 computeCapability: 8.6
coreClock: 1.837GHz coreCount: 28 deviceMemorySize: 12.00GiB deviceMemoryBandwidth: 335.32GiB/s
2021-04-29 20:58:55.660196: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll2021-04-29 20:58:56.114244: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll
2021-04-29 20:58:56.114406: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll
2021-04-29 20:58:56.411205: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll
2021-04-29 20:58:56.446609: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll
2021-04-29 20:58:56.679981: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll
2021-04-29 20:58:56.879028: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll
2021-04-29 20:58:56.899777: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll   
2021-04-29 20:58:56.900259: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-04-29 20:58:56.904263: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-04-29 20:58:56.906490: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce RTX 3060 computeCapability: 8.6
coreClock: 1.837GHz coreCount: 28 deviceMemorySize: 12.00GiB deviceMemoryBandwidth: 335.32GiB/s
2021-04-29 20:58:56.906595: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll2021-04-29 20:58:56.906673: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll 
2021-04-29 20:58:56.906756: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll
2021-04-29 20:58:56.906847: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll  
2021-04-29 20:58:56.906913: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll 
2021-04-29 20:58:56.906974: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll
2021-04-29 20:58:56.907042: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll
2021-04-29 20:58:56.907106: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll   
2021-04-29 20:58:56.907231: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0
2021-04-29 20:58:57.837122: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1261] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-04-29 20:58:57.837273: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1267]      0
2021-04-29 20:58:57.837423: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1280] 0:   N
2021-04-29 20:58:57.838305: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10491 MB memory) -> physical GPU (device: 0, name: GeForce RTX 3060, pci bus id: 0000:01:00.0, compute capability: 8.6)       
2021-04-29 20:58:57.840165: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
Loading and resizing images...
Loading and resizing images...
Compiling model...
Fitting model...

Epoch 1/10
52/52 [==============================] - 22s 264ms/step - loss: 0.7597 - accuracy: 0.7440 - val_loss: 0.2065 - val_accuracy: 0.9595
Epoch 2/10
52/52 [==============================] - 13s 255ms/step - loss: 0.1916 - accuracy: 0.9597 - val_loss: 0.1634 - val_accuracy: 0.9595
Epoch 3/10
52/52 [==============================] - 13s 250ms/step - loss: 0.0918 - accuracy: 0.9804 - val_loss: 0.1683 - val_accuracy: 0.9595
Epoch 4/10
52/52 [==============================] - 13s 252ms/step - loss: 0.0934 - accuracy: 0.9741 - val_loss: 0.2116 - val_accuracy: 0.9595
Epoch 5/10
52/52 [==============================] - 14s 261ms/step - loss: 0.0669 - accuracy: 0.9852 - val_loss: 0.1899 - val_accuracy: 0.9595
Epoch 6/10
52/52 [==============================] - 13s 245ms/step - loss: 0.0578 - accuracy: 0.9858 - val_loss: 0.2215 - val_accuracy: 0.9622
Epoch 7/10
52/52 [==============================] - 13s 258ms/step - loss: 0.0470 - accuracy: 0.9903 - val_loss: 0.1755 - val_accuracy: 0.9622
Epoch 8/10
52/52 [==============================] - 13s 255ms/step - loss: 0.0456 - accuracy: 0.9880 - val_loss: 0.1774 - val_accuracy: 0.9622
Epoch 9/10
52/52 [==============================] - 13s 257ms/step - loss: 0.0478 - accuracy: 0.9872 - val_loss: 0.1299 - val_accuracy: 0.9703
Epoch 10/10
52/52 [==============================] - 14s 257ms/step - loss: 0.0585 - accuracy: 0.9862 - val_loss: 0.1245 - val_accuracy: 0.9676
Testing accuracy...
              precision    recall  f1-score   support

    non-fire       1.00      0.20      0.33        15
        fire       0.97      1.00      0.98       355

    accuracy                           0.97       370
   macro avg       0.98      0.60      0.66       370
weighted avg       0.97      0.97      0.96       370

Saving model...
2021-04-29 21:01:54.340297: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.